apps/sample-workers-vite-react/README.md

```diff
@@ -0,0 +1,48 @@
+# Xata Workers Vite React Sample App
+
+This is a sample app that uses [Xata Workers](https://xata.io/beta/workers) and [Vite](https://vitejs.dev) to build a React app.
+
+## Getting Started
+
+1. Install dependencies
+
+```bash
+pnpm install
+```
+
+2. Connect to your Xata database (demo db)
+
+```bash
+xata init
+```
+
+3. Configure Xata Workers
+
+```bash
+xata workers init
+```
+
+4. Start the app
+
+```bash
+pnpm dev
+xata workers watch
+```
+
+## Deploying
+
+1. Deploy Xata Workers
+
+```bash
+xata workers upload
+```
+
+2. Build the app
+
+```bash
+pnpm build
+```
+
+## Xata Workers feedback
+
+We'd love to hear your feedback on Xata Workers. Please [join our Discord](http://xata.io/discord) and let us know what you think!
```

Comment:
<img width="330" alt="image" src="https://user-images.githubusercontent.com/1761469/204290709-4dc99c2b-7aa5-444e-b27f-62c459f8bb65.png">

So far, the application does nothing, and I have this as an error in my console.

Reply:
But this is probably because my database is empty, without any schema. Can you provide the schema and a script to populate the database like other example apps?

Reply:
It's the demo database schema.

Reply:
I tried with the demo db, I still have the same error (500 no fetch handler defined)

Reply:
Then I need to check it out, and see if I can replicate it locally

Reply:
Fabien didn't have the Workers Feature flag, I added it to him.